src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+if (logo) {
+  const messageDiv = document.createElement('div');
+
+  messageDiv.classList.add('message');
+
+  function handleLogoClick() {
+    setTimeout(() =>{
+    const promise1 = Promise.resolve('Promise was resolved!');
+
+      promise1.then((result) => {
+        messageDiv.innerText = result;
+        document.body.appendChild(messageDiv);
+      });
+    }, 0)
```

Comment:
You are creating a promise and immediately resolving it. This is not a common use case for promises. Promises are typically used for asynchronous operations where you don't know when the operation will complete. If you just want to delay the execution of code, you can just use `setTimeout` without a Promise.